build/tasks/LinuxPackageRepository/AddPackageStrategy.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I'm assuming this was copied from somewhere else? @Eilon  - license stuff

Reply:
From where was it copied?

Reply:
As per my original comment, this is copied from https://github.com/dotnet/cli/pull/7460. I didn't bother with updating the license headers since I assume this is only temporary and we will be moving the tool to dotnet/arcade soon?

Reply:
This is ok in this circumstance. Please consult me whenever copying code from any repo outside of our org (or product).

Reply:
Will do.